index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Isaac Codes</title>
+    <link rel="stylesheet" href="styles/styles.css">
+  </head>
+  <body>
+    <header>
+      <h2><a href="index.html">Isaac Codes</a></h2>
+      <nav>
+        <ul>
+          <li><a href="portfolio.html">Portfolio</a></li>
+          <li><a href="code-journal.html">Code Journal</a></li>
+          <li><a href="about.html">About</a></li>
+        </ul>
+      </nav>
+    </header>
+
+    <main>
+      <section id="intro">
+        <h1>Isaac Codes: The Journey Begins</h1>
+        <p> > You stumble upon the website of a budding web developer.</p>
```

Comment:
We didn't mention this in class, but something to think about in the future: your HTML will like you better if you use characters like the angle brackets in text as an HTML Entity. Read more about it here: https://www.w3schools.com/html/html_entities.asp